packages/plugins/visitor-plugin-common/src/utils.ts

```diff
@@ -150,13 +152,17 @@ export class DeclarationBlock {
       }
 
       if (this._methodName) {
-        result += `${this._methodName}({${this._config.blockWrapper}
-${this._block}
-${this._config.blockWrapper}})`;
+        const block = `{${this._config.blockWrapper}
```

Comment:
I think some has been reformatted here and now causes a lot of whitespaces.  @kamilkisiela

Reply:
yep, I'm `yarn build`ing the fix